### ISSUE TITLE
New filter button active styles being overridden by focus state

### ISSUE THREAD

**Issue by jrdn91** [blackbaud/skyux1#790]
### Expected behavior

The active styles for the filter button should always be present when passed property is true
### Actual behavior

Filter button active styles do not show up if the filter button has focus
### Steps to reproduce

Set the `bb-filter-button-active` to true and then give the `button` element inside of the `bb-filter-button` focus in the inspector.
### Plunker

http://plnkr.co/edit/JMnPH2kf8VXqWUbbMCuf?p=preview
Basically the basic plunker example from the docs with the `bb-filter-button-active` attribute

**Comment by Blackbaud-PatrickOFriel** [comment]
Hey @Jordan4jc please add a plunker as it gives us a live versioned example of this issue occurring.

**Comment by jrdn91** [comment]
Added a plunker, we notice this a bit more than in this example because I've actually removed all the "blue halo's" from focused buttons and inputs so you don't notice it until you've clicked off of the button thus un-focusing it

**Comment by Blackbaud-PatrickOFriel** [comment]
We can fix the border color/size issue. The color of the filter icon, however, is correct because that is the color text in that button should be when focused (also when hovered). 

I would suggest not removing all the "blue halos" from your application because showing focus indication is generally a good thing for accessibility and modern web design in general.

**Comment by jrdn91** [comment]
Right, unfortunately this decision wasn't up to me

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.2](https://github.com/blackbaud/skyux/releases/tag/v1.7.2)
